src/transformState.js

```diff
@@ -6,6 +6,38 @@
  */
 function transformState(state, actions) {
   // write code here
-}
+  function addProps(add) {
```

Comment:
```suggestion
  function addProps(propsToAdd) {
```
Let's use more descriptive names for function props